README.md

```diff
@@ -61,7 +61,6 @@ Various third parties have built tooling on top of packwerk. Here's a selection
 - https://github.com/Gusto/packwerk-vscode integrates packwerk into Visual Studio Code so you can see violations right in your editor
 - https://github.com/Gusto/stimpack sets up Rails autoloading, as well as `rspec` and `FactoryBot` integration, for packages arranged in a flat list. Stimpack is quite convenient, but for autoloading we recommend to use `Rails::Engine`s instead.
 - https://github.com/rubyatscale/danger-packwerk integrates packwerk with [danger.systems](https://danger.systems) to provide packwerk feedback as Github inline PR comments
-- https://github.com/rubyatscale/packwerk-extensions contains extensions for packwerk, including a checker for packwerk that allows you to enforce public API boundaries. This was originally extracted from `packwerk` itself.
```

Comment:
This still seems like it would be helpful to link as this is a section for surrounding ecosystem gems.

Reply:
Good point. It is under the ecosystem of gems, which doesn't necessarily mean people have to use it. I think the broader change is of removing it from usage is more important. Your gem can own the documentation for privacy checking since it is now responsible for it.

Reply:
`packwerk-extensions` owning documentation for privacy checking makes perfect sense to me 👍

Reply:
Yeah, I forgot we had this list already. If we have all those gems here, it is fine to have the privacy checking gem as well. Sorry for the confusion.